event_webhook_types.go

```diff
@@ -87,15 +87,15 @@ type BuildEvent struct {
 	ProjectName       string     `json:"project_name"`
 	User              *EventUser `json:"user"`
 	Commit            struct {
-		ID          int    `json:"id"`
-		SHA         string `json:"sha"`
-		Message     string `json:"message"`
-		AuthorName  string `json:"author_name"`
-		AuthorEmail string `json:"author_email"`
-		Status      string `json:"status"`
-		Duration    int    `json:"duration"`
```

Comment:
I cannot find this specific field in the docs. Can you show me something that shows this needs to be a `float64`?

Reply:
My bad, it was int. I tested in local.